(Doc+) link videos for allocation and ilm (#116880)
elastic/elasticsearch: docs/reference/snapshot-restore/repository-s3.asciidoc

*If you are looking for a hosted solution of Elasticsearch on AWS, please visit
https://www.elastic.co/cloud/.*

> See this https://www.youtube.com/watch?v=ACqfyzWf-xs[connecting an AWS S3 repository video]
> for an example setup walkthrough.

Reviewer: ```suggestion
See https://www.youtube.com/watch?v=ACqfyzWf-xs[this video]
for a walkthrough of connecting an AWS S3 repository.
```
